### DISCUSSION TITLE
Please I meet some problems

### DISCUSSION THREAD

**Discussion by haber1917** [Mikubill/sd-webui-controlnet#2803] [General]
![屏幕截图 2024-04-24 172540](https://github.com/Mikubill/sd-webui-controlnet/assets/164627352/3ab3061e-f315-4531-8801-746216a88d0e)
wen
![屏幕截图 2024-04-24 173015](https://github.com/Mikubill/sd-webui-controlnet/assets/164627352/ff508d0c-70e8-4f61-ae08-592bb313409c)
当我尝试把controlnet预处理后的图像导入提示词生成图片时，我的sd报错了。我不知道如何处理这个棘手的问题。When I try to import the controlnet preprocessed image into the prompt to generate the image, my SD error is reported. I don't know how to deal with this tricky issue.Please help me

**Comment by huchenlei** [comment]
Please file an issue following bug template and attach full console logs.
